e2e-tests/playwright/e2e/configuration-test/config-map.spec.ts

```diff
@@ -8,7 +8,7 @@ test.describe("Change app-config at e2e test runtime", () => {
     test.setTimeout(300000); // Increasing to 5 minutes
 
     const configMapName = "app-config-rhdh";
-    const namespace = "showcase-runtime";
+    const namespace = process.env.NAME_SPACE;
```

Comment:
NAME_SPACE is already set and updating it might create some confusions, may you can use the NAME_SPACE_RUNTIME

Reply:
@subhashkhileri I decided to remove from the k8's nightly, since it's already running for showcase-runtime on ibm cloud